bool  clockwise_forward # check if is clockwise or forward
---
